x-pack/plugins/observability/public/update_global_navigation.tsx

```diff
@@ -31,25 +28,19 @@ export function updateGlobalNavigation({
         return {
           ...link,
           navLinkStatus:
-            config.unsafe.cases.enabled && capabilities[casesFeatureId].read_cases && someVisible
+            capabilities[casesFeatureId].read_cases && someVisible
               ? AppNavLinkStatus.visible
               : AppNavLinkStatus.hidden,
```

Comment:
Arguably, this is out of this PR scope, but the condition `someVisible ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden` has been repeated 3 times in the page. @mgiota, I leave it to you whether you want to refactor it within this PR